### ISSUE TITLE
strange rm directory error running libinput-gestures-setup

### ISSUE THREAD

**Issue by wolftune** [bulletmark/libinput-gestures#217]
Before creating a new issue, please follow each step in the TROUBLESHOOTING section
of the main README.

```
Run libinput-gestures -l on the command line and paste the output here.

libinput-gestures: session KDE+x11 on Linux-4.18.0-20-lowlatency-x86_64-with-neon-18.04-bionic, python 3.6.7, libinput 1.10.4
/usr/bin/libinput-gestures: hash 34ab6c1e45b5673bf2b143a6321c81c4
Gestures configured in /etc/libinput-gestures.conf:
swipe up           _internal ws_up
swipe down         _internal ws_down
swipe left         xdotool key alt+Right
swipe right        xdotool key alt+Left
pinch in           xdotool key super+s
pinch out          xdotool key super+s
libinput-gestures: device /dev/input/event7: Apple Wireless Trackpad
```

**Describe the issue**
A clear and concise description of what the issue is.

I followed the install instructions carefully. But whenever I run libinput-gestures-setup I get a weird rm error along with the rest of the expected output, for example:

```
~$ libinput-gestures-setup stop
rm: cannot remove '/home/aaron/bin/libinput-gestures': Is a directory
libinput-debug-events stopped.
libinput-gestures stopped.
```

**Comment by bulletmark** [comment]
It seems you have some old junk you must have created some time ago in a directory `/home/aaron/bin/libinput-gestures`. Do you remember creating that dir?

**Comment by wolftune** [comment]
I only just now made the directory by doing a git clone command in /home/aaron/bin/

**Comment by bulletmark** [comment]
By convention you would normally keep only personal executable programs/scripts in `$HOME/bin` and then add that to your `PATH`. You should not really keep dirs and/or source code in there. Remove that dir and then do your `git clone` in your home dir. I will add a change to that setup script to ignore any error if this happens again because users should be free to put stuff wherever they want of course.

**Comment by wolftune** [comment]
Thanks, I sorta best-guess figured that ~/bin/ was a good place for stuff-that-wasn't-from-package-manager, assuming it ended up with binaries in that directory or something.

Is there a reason to not just *move* the directory but instead to remove and re-clone?

**Comment by bulletmark** [comment]
That's a git question but no, you can move a git repo dir anywhere and not affect anything. I guess that may save you about 1 sec over just re-cloning it again?! ;)

**Comment by wolftune** [comment]
I know it's not a git issue. I was asking after surprise about your suggestion. I wondered if perhaps somehow there was some issue about the location from which I did the `make install`, but that would be surprising too. Sorry for the confusion, and thanks for clarifying.

**Comment by bulletmark** [comment]
No it does not matter where you do the make install. The problem is that the very early versions of this program used to install locally only for the user doing the make to his/her `~/bin/` but long ago I changed it to install globally and at that time I added some code to clean up any old versions left in `~/bin` and some other personal places. That logic is what was trying to delete your git dir. I have fixed that logic to not complain in the future and will submit that change soon but my suggestion to you also is that is not a good place to put your source/git dir anyhow.
